csharp/AppEncryption/AppEncryption/Persistence/AdoMetastoreImpl.cs

```diff
@@ -14,6 +14,11 @@
 
 namespace GoDaddy.Asherah.AppEncryption.Persistence
 {
+    /// <summary>
+    /// Provides an Ado based implementation of <see cref="IMetastore{T}"/> to store and retrieve
+    /// <seealso cref="JObject"/> values. It uses the table name "encryption_key" to perform all RDBMS based operations.
```

Comment:
JObject values is very generic. The metastore is specifically used to store and retrieve system and intermediate key values. Should be changed accordingly.